src/fidesops/service/masking/strategy/masking_strategy_aes_encrypt.py

```diff
@@ -16,13 +16,17 @@
 )
 from fidesops.service.masking.strategy.format_preservation import FormatPreservation
 from fidesops.service.masking.strategy.masking_strategy import MaskingStrategy
+from fidesops.service.masking.strategy.masking_strategy_factory import (
+    MaskingStrategyFactory,
+)
 from fidesops.util.encryption.aes_gcm_encryption_scheme import encrypt
 from fidesops.util.encryption.hmac_encryption_scheme import hmac_encrypt_return_bytes
 from fidesops.util.encryption.secrets_util import SecretsUtil
 
 AES_ENCRYPT = "aes_encrypt"
```

Comment:
The naming here doesn't really imply why this value exists. It would be more straightforward if it were slightly more verbose, like `AES_ENCRYPT_STRATEGY_NAME` 🤷‍♂️ 

This applies to the other masking strategies as well.

Reply:
updated in https://github.com/ethyca/fidesops/pull/560/commits/0abfd85ddddc190128c2bc5cbc5e67b1565f58a2